controllers/PasswordController/resetFromUser.js

```diff
@@ -0,0 +1,44 @@
+/*eslint-disable*/
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const User = require('../../models/Users');
+
+/**
+ * @description recieves a token and a new password from the user, verifies the token and updates the user's Password
+ * @param {object} req - request object
+ * @param {object} res - response object
+ */
+const resetByUser = (req, res) => {
+  const { token, password } = req.body;
```

Comment:
no validation. use joi.

Reply:
On it

Reply:
I have deleted the file sapphire already implemented it